torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -362,19 +362,6 @@ Val* getMaximumValue(DataType v) {
 
 } // namespace
 
-// TENSOR FACTORIES
-TensorView* rand(const std::vector<Val*>& shape, DataType dtype) {
```

Comment:
moved below, just some code organization change in this file, no real behavioral change.